src/SPECS/hootenanny.spec.in

```diff
@@ -18,7 +18,8 @@ BuildRequires:  git glpk-devel graphviz libicu-devel
 BuildRequires:  hootenanny-words = 1.0.0
 BuildRequires:  libspatialite-devel
 BuildRequires:  log4cxx-devel nodejs-devel opencv-devel
-BuildRequires:  postgresql-devel >= 9.1
+#BuildRequires:  postgresql-devel >= 9.1
+BuildRequires:  postgresql93-devel
```

Comment:
Ah, good point about already running 9.3.  I'm all about consolidating versions so if we think 9.3 is the way to go, then I'm all for it.  Just want to make sure we've thought about any impacts to upgrading.